data-prepper-plugins/avro-codecs/src/test/java/org/opensearch/dataprepper/avro/AvroAutoSchemaGeneratorTest.java

```diff
@@ -218,7 +218,7 @@ static class SomeUnknownTypesArgumentsProvider implements ArgumentsProvider {
         @Override
         public Stream<? extends Arguments> provideArguments(ExtensionContext context) {
             return Stream.of(
-                    arguments(Random.class),
```

Comment:
In Java 21 (and maybe 17?), `Random` is `final` and this causes Mockito to fail when it mocks it. So, we test on the `Timer` class instead.